server/src/modules/mentorship-request/mentorship-request.model.ts

```diff
@@ -41,6 +45,9 @@ export class MentorshipRequest extends Model<MentorshipRequest> {
   @Column({ type: DataType.DATE, allowNull: false })
   endDate: string;
 
+  @Column({ type: DataType.ENUM(...Object.values(RequestStatus)) })
+  requestStatus: RequestStatus;
+
```

Comment:
Small suggestion: previously we thought this wasn't necessary because you could just see if `mentorshipResponse` exists: is that still true? If it's more convenient to have `requestStatus` that's probably fine, but it does duplicate data.